src/_includes/_layouts/Home.jsx

```diff
@@ -121,19 +121,26 @@ export default ({ comp, title }) => {
             redirect_url={"projects"}
           />
         </section>
-        <section className="h-[17rem] bg-primary text-negative flex flex-col justify-evenly items-center justify-center py-16 max-lg:h-fit max-lg:gap-y-16 max-lg:py-16">
-          <h1 className="lg:text-5xl text-6xl mb-12">
-            <strong>Sponsors and Partners</strong>
+        <section
+          className="bg-negative text-negative flex flex-col justify-evenly  py-16 max-lg:h-fit max-lg:gap-y-6 max-lg:py-16"
```

Comment:
I think we could add some horizontal padding to this section (to have it align with the padding of the right section of the hero page so using `px-20`). It just helps with making the page look more professional and presentable to these NPOs as this home page could be their first time interacting with our group yknow.